InvenTree/stock/models.py

```diff
@@ -1290,6 +1290,8 @@ def __str__(self):
         if self.location:
             s += ' @ {loc}'.format(loc=self.location.name)
 
+        s += ' (PO{po})'.format(po=self.purchase_order)
```

Comment:
Instead of hard-coding "PO" here, please look at the `PURCHASEORDER_REFERENCE_PREFIX` setting (in the `common.models` file). The purchase order prefix is user configurable